packages/cli/src/commands/doctor/healthchecks/__tests__/androidHomeEnvVariable.test.ts

```diff
@@ -0,0 +1,56 @@
+import androidHomeEnvVariables from '../androidHomeEnvVariable';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {Ora} from 'ora';
+
+jest.mock('../common');
+
+import {logManualInstallation} from '../common';
+
+describe('androidHomeEnvVariables', () => {
+  const OLD_ENV = process.env;
+
+  it('returns a message if no ANDROID_HOME is defined', async () => {
+    delete process.env.ANDROID_HOME;
+
+    const environmentInfo: EnvironmentInfo = JSON.parse(
+      await getEnvironmentInfo(),
+    );
+    const diagnostics = await androidHomeEnvVariables.getDiagnostics(
+      environmentInfo,
+    );
+    expect(typeof diagnostics.needsToBeFixed).toBe('string');
+  });
+
+  it('returns false if ANDROID_HOME is defined', async () => {
+    process.env.ANDROID_HOME = '/fake/path/to/android/home';
+
+    const environmentInfo: EnvironmentInfo = JSON.parse(
+      await getEnvironmentInfo(),
+    );
+    const diagnostics = await androidHomeEnvVariables.getDiagnostics(
+      environmentInfo,
+    );
+    expect(diagnostics.needsToBeFixed).toBe(false);
+  });
+
+  it('logs manual installation steps to the screen', async () => {
+    // @ts-ignore
```

Comment:
Why do you need this?

Reply:
Because I'm forcing `loader` to be of type `Ora`, but I only put the `info` property, not the rest.

Though I changed it to `any` to avoid the `// @ts-ignore`.

Reply:
You can use `NoopLoader` (grep for it)

Reply:
I need to watch it though, so I need to use `jest.fn()`. Though using `NoopLoader`, I could use `jest.spyOn()`. What do you reckon is the best?

Reply:
Generate the mock: https://jestjs.io/docs/en/jest-object#jestgenmockfrommodulemodulename

Reply:
Actually my mistake, I don't need to watch it, so I used the `NoopLoader`. Thanks @thymikee!